### DISCUSSION TITLE
Support Sidekiq 6 only?

### DISCUSSION THREAD

**Discussion by ThomasKoppensteiner** [ThomasKoppensteiner/sidekiq-undertaker#39] [Q&A] [answered]
Sidekiq `6.0.0` was released in [August 2019](https://rubygems.org/gems/sidekiq/versions).
I'm thinking about dropping the support for earlier Sidekiq versions.

With which Sidekiq version are use using Sidekiq::Undertaker?

@markkocera, @nilsding, @alexJunger, @diendorfersarah, @biow0lf, @agustin, @kammerer, @zoras, @majksner, @normancapule, @dmitry and @xuwupeng2000wework you all "stared" this project, that is why I want to ask you if you still need support for Sidekiq 5 or if support for Sidekiq 6 would be enough for you?

**Comment by nilsding** [comment]
Sidekiq 6 or newer is fine by me.

**Reply by kammerer** [reply]
Same for me. Thanks!

**Comment by ThomasKoppensteiner** [comment, marked as answer]
Done. Details see CHANGELOG
